specification/securityinsights/resource-manager/common/1.0/types.json

```diff
@@ -35,6 +35,25 @@
       "type": "object",
       "x-ms-external": true
     },
+    "EntityCommonProperties": {
+      "description": "Entity common property bag.",
+      "properties": {
+        "additionalData": {
```

Comment:
>additionalData [](http://example.com/codeflow?start=9&length=14)

ARM recommends against the use of the additionalProperties pattern. Open-ended unschematized objects are problematic, even though they appear to make some things easier for RP owners. ARM is evolving toward stronger typing wherever possible because it allows all consumers of your API to reason about it. This has benefits for everyone that interacts with your API from infrastructure, to other Azure features to SDK owners to customers. Many APIs use unschematized objects, but it's a pattern that should be avoided if at all possible.

For example, customers will not be able to manage these objects using Azure Policy, which is a standard feature. Your customers may be unhappy about that. There are other features like ARG that won't be able to support these objects, and others features and capabilities in the pipeline.

Reply:
Previously reviewed.